flow/connectors/postgres/client.go

```diff
@@ -523,7 +523,7 @@ func (c *PostgresConnector) jobMetadataExists(ctx context.Context, jobName strin
 	return result.Bool, nil
 }
 
-func majorVersion(ctx context.Context, conn *pgx.Conn) (PGVersion, error) {
+func GetMajorVersion(ctx context.Context, conn *pgx.Conn) (PGVersion, error) {
```

Comment:
Could move this function to `shared/postgres.go`

Reply:
#1576